### DISCUSSION TITLE
Do I need to install firetools+firejail or just firetools?

### DISCUSSION THREAD

**Discussion by Big-Ug** [netblue30/firejail#4880] [Q&A] [answered]
Hi gang,

  I'm not a coder, in any sense of the term. I'm a plain and simple Linux user, who, understands just enough to mess my system up once in a while because I like to try and do things that people with good memory and intelligence do easily, heh, like upgrading a kernel and still not understanding how to do the whole initrd thing and having to reinstall the whole OS, lol. (I use Slackware, btw, have been since 12.2, if I remember right) 

  So, because even I, as slow as I am, am seeing browsers getting worse and worse about going the way 'Microslop' and such, to the point that websites don't hardly let my preferred browsers work very well if at all, and Firefox is getting too big for its britches and trying to mimic Google ('We are bigger than government, give us all your passwords and personal info so we can watch you, peon!') etc.

  If I install Firetools, does that also contain 'Firejail'? Or am I supposed to install Firejail and *then* install Firetools for the GUI use of it?

  Thanks for putting up with my blathering just to ask a couple of simple, dumb questions!

**Comment by reinerh** [comment, marked as answer]
You need to install both. `firetools` is just the graphical frontend for `firejail`, that simplifies starting jailed applications, has a few monitoring tools built-in, etc.
